### ISSUE TITLE
Web Components: Card section - simple - beginning and ending boundary for the card is not conveyed for the screen reader user. 

### ISSUE THREAD

**Issue by RobertaJHahn** [carbon-design-system/carbon-for-ibm-dotcom#6005]
## Detailed description

Issue: 
1. The beginning and ending boundary for the card is not conveyed for the screen reader user. 
2. In tab / shift tab navigation, when focus reaches the link in each card, Screen reader announces a lengthy text which makes it oververbose.

Expected:
1. Provide each card within the landmark region and give appropriate label.
2. It may be possible if the entire content in the card is given in anchor tag. In this scenario, only the heading can be given in anchor tag. Note: the link text/label should be concise and convey only the relevant information. 

Note: working fine on react version.

## Additional information
- this fix should address the bug included in issue (#5504 ). Engineer should check to be sure this fix addresses the bug included in (#5504 )
 - Reach out to kenny for additional details.
- The React bug was closed in sprint 21-09 and included in the May v1.19.0 release under issue (#4613 ).  Assigning to Ari as she worked on fixing the React bug.

**Comment by Praveen-sr** [comment]
@ariellalgilmore Ram re tested this on both JAWS and Voiceover on the [ link ]( https://ibmdotcom-webcomponents.s3.us-east.cloud-object-storage.appdomain.cloud/deploy-previews/6642/index.html?path=/story/overview-getting-started--page)  - Working as expected.
